sdk/src/test/scala/com/horizen/api/http/SidechainTransactionApiRouteTest.scala

```diff
@@ -338,7 +338,7 @@ class SidechainTransactionApiRouteTest extends SidechainApiRouteTest {
       Post(basePath + "decodeTransactionBytes")
         .withEntity(SerializationUtil.serialize(ReqDecodeTransactionBytes("AAABBBCCC"))) ~> sidechainTransactionApiRoute ~> check {
         status.intValue() shouldBe StatusCodes.InternalServerError.intValue
-        responseEntity.getContentType() shouldEqual ContentTypes.`application/json`
+        responseEntity.getContentType() shouldEqual ContentTypes.`text/plain(UTF-8)`
```

Comment:
We are hitting a default here, cause for error 500 we do not wrap it somewhere in the code.
```
  implicit def multipartFormDataUnmarshaller(implicit log: LoggingAdapter = NoLogging, parserSettings: ParserSettings = null): FromEntityUnmarshaller[Multipart.FormData] =
    multipartUnmarshaller[Multipart.FormData, Multipart.FormData.BodyPart, Multipart.FormData.BodyPart.Strict](
      mediaRange = `multipart/form-data`,
      defaultContentType = ContentTypes.`text/plain(UTF-8)`,
      createBodyPart = (entity, headers) => Multipart.General.BodyPart(entity, headers).toFormDataBodyPart.get,
      createStreamed = (_, parts) => Multipart.FormData(parts),
      createStrictBodyPart = (entity, headers) => Multipart.General.BodyPart.Strict(entity, headers).toFormDataBodyPart.get,
      createStrict = (_, parts) => Multipart.FormData.Strict(parts))
```

Are there any expectations/requirements to return some exact ContentType in case of error 500?
For me, it seems to be redundant

Reply:
no, it is fine. In case of 500, no content expected at all.